src/Components/Footer/Footerlink/index.js

```diff
@@ -0,0 +1,13 @@
+import React, {Component} from 'react';
+import './index.css';
+
+
+    export default class Footerlink extends Component {
```

Comment:
Maybe you could keep the ident consistent here.

Reply:
doing it